src/python/gudhi/cubical_complex.pyx

```diff
@@ -191,6 +221,11 @@ cdef class CubicalComplex:
         coordinates to the corresponding value in the input (i.e. the filtration values of
         the top-dimensional cells) is useful for differentiation purposes.
 
+        Since the cubical complex construction from vertices is different from the top dimensional one,
+        the former may not have an equivalent with the second construction, and vice versa.
+        Therefore, using cofaces_of_persistence_pairs with a cubical complex constructed from vertices
+        can lead to an undefined behavior.
```

Comment:
Hum I don't think that the undefined behavior occurs necessarily when constructing with vertices and calling `cofaces_of_persistence_pairs` (and vice versa). I think that it can only lead to a problem if a given cubical complex cannot be constructed with the two different ways (using vertices or top cells) - maybe that's always the case? (to be confirmed @mglisse)
But in the case the user knows what he is doing, we shouldn't prevent him from calling what he wants...